server/src/database/mod.rs

```diff
@@ -0,0 +1,35 @@
+use postgres::{Connection, TlsMode, Error};
+use std::borrow::Borrow;
+use std::{thread, time};
+use super::utils::get_env_variable;
```

Comment:
Do you think we can replace by `use crate::utils::get_env_variable;`? I think chances are `utils` will stay at the root of the project, while I'm not sure `database` and `utils` will stay at the same level.